include/seastar/core/scheduling.hh

```diff
@@ -293,6 +293,13 @@ public:
     friend unsigned internal::scheduling_group_index(scheduling_group sg) noexcept;
     friend scheduling_group internal::scheduling_group_from_index(unsigned index) noexcept;
 
+    struct stats {
+        std::chrono::duration<float> runtime;
```

Comment:
Actually, why are these floats?

Reply:
I'm not super experienced with the chrono api. Isn't this the idiodmatic to report "fractional seconds"? I guess the alternative is to return nanos or milis, but I don't know what the idiomatic way of doing a non-narrowing conversion is in that case.

Reply:
Oh, I see. I guess the assignment is converting to seconds. I'd be tempted to not do the conversion here, and leave them as std::chrono::nanoseconds.

Reply:
Lets export `sched_clock::duration` so that no conversion happens in seastar, and then convert when we read it in Redpanda.

Reply:
Done.